Makefile.common

```diff
@@ -375,7 +376,8 @@ ifeq ($(HAVE_LIGHTREC), 1)
                   $(DEPS_DIR)/lightrec/memmanager.c \
                   $(DEPS_DIR)/lightrec/optimizer.c \
                   $(DEPS_DIR)/lightrec/reaper.c \
-                  $(DEPS_DIR)/lightrec/regcache.c
+                  $(DEPS_DIR)/lightrec/regcache.c \
+                  $(DEPS_DIR)/lightrec/tlsf/tlsf.c
```

Comment:
You shouldn't have to compile TLSF since you disabled the code buffer support (ENABLE_CODE_BUFFER=0)

Reply:
ENABLE_CODE_BUFFER=1 was set in 3012b97d6e2ec3bcd6cec4f8ed7e9bf22f568ef8 of this PR, there were many changes squashed together, and it could maybe use even more to make this more clear maybe?

Reply:
Yes I saw it after my message. I guess this change should be squashed into 3012b97 then.